### PR TITLE
Angle collateral and perp events-calls

### PR DESCRIPTION
**Pull request by AyadiGithub**
## What?
Add Angle.money collateral and perpetual events-calls

## How? 
I used [abi-parser](https://nansen-contract-parser-prod.web.app/) to build the table definitions

### CODE REVIEW

File: dags/resources/stages/parse/table_definitions/angle/StableMasterFront_call_mint.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "minStableAmount",
+                "type": "STRING"
+            }
+        ],
+        "table_description": "",
+        "table_name": "StableMasterFront_call_event_mint"
```
Comment (medvedev1088) [inline dags/resources/stages/parse/table_definitions/angle/StableMasterFront_call_mint.json:60]: Should be StableMasterFront_call_mint

Comment (AyadiGithub) [reply dags/resources/stages/parse/table_definitions/angle/StableMasterFront_call_mint.json]: fixed. Check still fails on contract address field: 
SELECT '0x5efe48f8383921d950683c46b87e28e21dea9fb5' UNION ALL SELECT '0x98fdbc5497599eff830923ea1ee152adb9a4cea5' UNION ALL SELECT '0x4121a258674e507c990cdf390f74d4ef27592114' UNION ALL SELECT '0xb924497a1157b1f8835c93cb7f3d4aa6d2f227ba' UNION ALL SELECT '0xfc8f9eefc5fce1d9dace2b0a11a1e184381787c4'

Comment (araa47) [reply dags/resources/stages/parse/table_definitions/angle/StableMasterFront_call_mint.json]: What if you use this syntax instead: https://github.com/blockchain-etl/ethereum-etl-airflow/blob/425ebdbb573fb4db2842ede571ef79f3f4d07298/dags/resources/stages/parse/table_definitions/synthetix/sBTC_event_SelfDestructTerminated.json#L9

Comment (AyadiGithub) [reply dags/resources/stages/parse/table_definitions/angle/StableMasterFront_call_mint.json]: Worked! thank you @araa47

**Comment by SimonCousaert** [comment]
@araa47 @TimNooren Can you please check this PR? Greatly appreciated, thank you.

**Review by TimNooren — APPROVED** [review]

**Pull request merged**